data/csp/sle15/sp7/defaults.yaml

```diff
@@ -0,0 +1,8 @@
+preferences:
+  type:
+    _attributes:
+      image: oem
+packages:
+  _namespace_default_oem_repart:
+    package:
+      - dracut-kiwi-oem-repart
```

Comment:
Do we have enough time to sort out all the potential conflicts and image clean ups? We will need to disable the cloud-init disk growing mechanism for AWS and Azure and drop the stuff from the GCE images that is based on the utils package growpart-rootgrow.

So there is a lot more work than just adding the kiwi dracut definition. I am fine having the follow up work in a separate PR.

Reply:
I didn't include the dracut Kiwi resize config part so I think the kiwi resizer should not be used (@schaefi please intervene if that is wrong). The reason why I included the package is because using image type `oem` requires it and for schema version 7.5 we need image type `oem`. And the reason for using schema version 7.5 is that it is the native version in SLE 15 and anything older than 7.4 (I think) is not supported anymore by upstream kiwi which makes generating image descriptions with recent kiwi installed a PITA.

Since the descriptions get converted into a 7.5 schema description anyway by kiwi I figured we might want to convert our image definitions in keg to use 7.5, and starting off with the SP7 ones seems like a good trial run to me.

Whether we want to use the kiwi resizer instead of the framework specific implementations is a different question.

Reply:
If you need an example kiwi description that switches the resize off for an oem build you can take a look here

* https://github.com/OSInside/kiwi/blob/main/build-tests/x86/tumbleweed/test-image-vagrant/appliance.kiwi

Reply:
I am fine using the kiwi resize feature we just need to get rid of the other implementations then. And we'll find out if someone really cares about how we resize later. I cannot think of a way to figure this out before the switch.

Reply:
If you consider the kiwi resizer in the system you need to add a /etc/dracut.conf.d/xxx.conf which keeps the kiwi resize dracut module in the initrd such that a kernel update which calls dracut again keeps the resizer active. If we say adding this config file is something we can live with than using the kiwi resizer should be good.

It has one advantage. The kiwi resizer runs in the initrd and when the rootfs is not yet mounted. The resize therefore has no hard dependencies against other ares of the system and should not be impacted by updates on ignition, growpart, cloud-init, etc. If it proves to work once it will continue to work unless we break kiwi

Reply:
The scheduling says that Beta2 is only 2 weeks away, Nov. 28th, which is when we usually try to have the images ready. Beta3 is Jan 23rd.

I am in favor of making the switch to the kiwi resize mechanism, ignore the self imposed beta2 deadline and give us the extra time to clean up growpart-rootgrow and the cloud-init stuff. @aosthof can you live with beta3 as the deadline to have the images ready? 

We'll need to do increased testing on the resize mechanism.

Reply:
@rjschwei yes I'm ok with that. That said we need to make sure to have images ready for testing by Beta 3 - release is planned for 2025-01-23. As the initial setup of the images in IBS takes some time until everything works as expected it would be great to have at least a first image description available in Devel:PubCloud:CrossCloud before Christmas.

Reply:
@schaefi could you please work with @smarlowucf to make sure we get the necessary setting to keep the repart in the initrd into our new $CSP-dracut-conf packages. Thanks.